content/mentee_application.md

```diff
@@ -0,0 +1,78 @@
+# Our Mentee Application Template
```

Comment:
Can you add the metadata on the top of both Markdown files as the other Markdown files? Thanks

Reply:
> The building on netlify is failing. I looked at the logs: `Could not generate HTML for "/mentee-application/"
> 
> I checked the [Gridsome doc](https://gridsome.org/docs/pages/):
> 
> * The Vue page should be named `MenteeApplication,vue` and `MentorApplication.vue`
>   
>   * the generated pages will be `mentee-application` and `mentor-application`
> * The Markdown should be named `mentee-application.md`  and `mentor-application.md`
> 
> Could you rename the files this way and also the links?

Thanks so much for looking into that. It is perfecly working now. :)